### PR TITLE
cephfs: replace with a different variable name for builtin `cap` attribute

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: internal/csi-common/driver.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/container-storage-interface/spec/lib/go/csi"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
-	klog "k8s.io/klog/v2"
```
Comment (Madhu-1) [inline internal/csi-common/driver.go:25]: why this is changed?

File: internal/cephfs/util.go [hunk 2]
```diff
@@ -67,8 +67,8 @@ func (cs *ControllerServer) validateCreateVolumeRequest(req *csi.CreateVolumeReq
 		return status.Error(codes.InvalidArgument, "volume Capabilities cannot be empty")
 	}
 
-	for _, cap := range reqCaps {
```
Comment (Madhu-1) [inline internal/cephfs/util.go:70]: instead of full name, I would like to go with `c`

**Review by Madhu-1 — CHANGES_REQUESTED** [review]

File: internal/csi-common/driver.go (hunk 1 above)
Comment (humblec) [reply internal/csi-common/driver.go:25]: the package naming is not required. Its redundant, however I could put it in a seperate commit.

File: internal/cephfs/util.go (hunk 2 above)
Comment (humblec) [reply internal/cephfs/util.go:70]: Why? with `capability` its explicit and clear what we are ranging on.

File: internal/csi-common/driver.go (hunk 1 above)
Comment (humblec) [reply internal/csi-common/driver.go:25]: @Madhu-1 do you want me to put it in a seperate commit?

Comment (Madhu-1) [reply internal/csi-common/driver.go:25]: This alias is present in other places also. I don't see a reason to change it unless some standard says we don't need to add an alias

File: internal/cephfs/util.go (hunk 2 above)
Comment (Madhu-1) [reply internal/cephfs/util.go:70]: https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Comment (humblec) [reply internal/cephfs/util.go:70]: `More unusual things and global variables need more descriptive names` -> this is the case here, Isnt it? I think thats why pick `cap` in first place. 

@nixpanic whats your thought here? 
My preference goes with `capability` though to make it clear.

Comment (nixpanic) [reply internal/cephfs/util.go:70]: `c` is fine, `req.VolumeCapabilities` is very explicit, and `ReqCaps` does not have the full name either, assuming the reader already knows what `Caps` refers to.

But, `capability` isn't an issue for me.

File: internal/csi-common/driver.go (hunk 1 above)
Comment (nixpanic) [reply internal/csi-common/driver.go:25]: I do not see any references changed from `klog` to `v2`. This changes looks unrelated to the renaming f the `cap` variable and should be in a different PR.

**Review by nixpanic — CHANGES_REQUESTED** [review]

File: internal/cephfs/util.go (hunk 2 above)
Comment (humblec) [reply internal/cephfs/util.go:70]: Considering we do have assignments like https://github.com/ceph/ceph-csi/pull/1600/files#diff-9eb942b29a818285935a4ffb1f6584a4d9e86317fbaf07a40a5029b35c2713b7R77 below , keeping it on `capability`. If really needed I can change to `c`.

File: internal/csi-common/driver.go (hunk 1 above)
Comment (humblec) [reply internal/csi-common/driver.go]: removed `klog` removal from this comment. That can be a separate PR altogether.

**Review by nixpanic — APPROVED** [review]

**Comment by nixpanic** [comment]
@mergifyio rebase

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**